pkg/controller/flightctl/flightctl.go

```diff
@@ -54,45 +50,92 @@ type FlightCtlManager struct {
 	recorder     events.Recorder
 
 	flightctlClient         flightctlClient
+	flightctlServer         string
 	agentRegistrationServer string
 
 	cachedToken string
 }
 
-func (f *FlightCtlManager) ApplyResources(ctx context.Context) error {
-	var err error
+// StartReconcileFlightCtlResources starts a loop to continuously reconcile FlightCtl resources
+func (f *FlightCtlManager) StartReconcileFlightCtlResources(ctx context.Context) {
```

Comment:
@qiujian16 @elgnay Change to reconcile the flightctl resources every 5 mins. No crash when error happens.